nbgrader/apps/assignapp.py

```diff
@@ -1,3 +1,5 @@
+# coding: utf-8
```

Comment:
I think these lines aren't strictly necessary since this will only make a difference if the special characters are in the python source code itself, and I don't think this is the case anywhere in nbgrader. But I don't think it hurts to have these lines, so it's fine for them to be there.

Reply:
Ah, I was uncertain about this myself, thanks for the feedback!